src/layout/GenericComponent.tsx

```diff
@@ -112,6 +114,7 @@ function ActualGenericComponent<Type extends CompTypes = CompTypes>({
     if (targetNode.item.id !== id) {
       return undefined;
     }
+    clearSearchParam(SearchParams.FocusComponentId);
```

Comment:
This might not be necessary if it happens in the `<Form />` component as well? :thinking:

Reply:
You're right. There was some timing issues with providing the `clearSearchParam` function from the `useNavigationParams` hook. Removed the function and cleared the parameters directly in `HandleNavigationFocusComponent` instead.